static/app/components/events/autofix/autofixMessageBox.tsx

```diff
@@ -51,14 +52,12 @@ interface AutofixMessageBoxProps {
   allowEmptyMessage: boolean;
   displayText: string;
   groupId: string;
-  inputPlaceholder: string;
   isDisabled: boolean;
-  onSend: ((message: string) => void) | null;
+  onSend: ((message: string, param?: boolean) => void) | null;
```

Comment:
```suggestion
  onSend: ((message: string, isCustom?: boolean) => void) | null;
```
can just state the boolean name here will be easier to follow